src/js/control-bar/mute-toggle.js

```diff
@@ -27,18 +31,18 @@ vjs.MuteToggle = vjs.Button.extend({
   }
 });
 
-vjs.MuteToggle.prototype.createEl = function(){
-  return vjs.Button.prototype.createEl.call(this, 'div', {
+MuteToggle.prototype.createEl = function(){
+  return Button.prototype.createEl.call(this, 'div', {
     className: 'vjs-mute-control vjs-control',
     innerHTML: '<div><span class="vjs-control-text">' + this.localize('Mute') + '</span></div>'
```

Comment:
For some of these template string references we should really just clean up the logic and store the inner-most element as the `contentEl_`, so we can just do `this.contentEl_.innerHTML = this.localize('Mute')`. That would help improve [line 57](https://github.com/videojs/video.js/pull/1976/files#diff-c5678798938810795d9c25bc9e2236ecL57) of this file... :expressionless: 
```
if(this.el_.children[0].children[0].innerHTML!=this.localize('Unmute')){
```